js/src/components/run/RunResultPane.tsx

```diff
@@ -144,16 +150,32 @@ const RunResultShareMenu = ({
               Share to Cloud
             </Button>
           ) : (
-            <Tooltip label="Please copy api token and relaunch recce with token">
-              <Button
-                leftIcon={<ExternalLinkIcon />}
-                size="sm"
-                onClick={() => {
-                  window.open("https://cloud.datarecce.io/settings#tokens", "_blank");
-                }}>
-                Enable sharing
-              </Button>
-            </Tooltip>
+            <Popover trigger="hover" placement="bottom-start">
```

Comment:
The component `<MenuItem>` is already clickable; we don't need to add a button inside the MenuItem.

Reply:
You can reference what we do in the `Explore` menu button.
https://github.com/DataRecce/recce/blob/b516928c5d1fe5b3ed6fd2eb51fa9002cef35e74/js/src/components/lineage/NodeView.tsx#L101

Reply:
Hi Kent,
As discussion, I will have another story for showing the Enable sharing button.
And I removed `Button` and used plain `MenuItem`.